plugins/hydra_submitit_launcher/hydra_plugins/hydra_submitit_launcher/submitit_launcher.py

```diff
@@ -142,7 +149,12 @@ def launch(
             )
 
         jobs = executor.map_array(self, *zip(*job_params))
-        return [j.results()[0] for j in jobs]
+
+        if self.no_block:
+            sentinal = JobReturn(status=JobStatus.COMPLETED, _return_value=None)
+            return [sentinal] * num_jobs
```

Comment:
Small typo, same in the 2 other files
```suggestion
            sentinel = JobReturn(status=JobStatus.COMPLETED, _return_value=None)
            return [sentinel] * num_jobs
```